src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml

```diff
@@ -3,33 +3,466 @@
+                            </Popup>
+                        </Grid>
+
+                        <Grid Margin="10,10,10,20" HorizontalAlignment="Stretch">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="Auto" />
+                            </Grid.ColumnDefinitions>
+
+                            <Button HorizontalAlignment="Right" Content="Cancel" Grid.Column="0" Style="{StaticResource GitHubVsButton}"/>
+                            <Button HorizontalAlignment="Right" Margin="6,0,0,0" Content="Create pull request" Grid.Column="1" Style="{StaticResource GitHubVsPrimaryActionButton}" />
+                        </Grid>
+                    </StackPanel>
+                </ScrollViewer>
+            </TabItem>
+
+            <TabItem Header="Commits" Style="{DynamicResource GitHubPRDetailsTabItem}">
+                <ListView Background="Transparent"
+                          BorderBrush="Transparent">
+                    <ListViewItem Style="{StaticResource CommitListItemContainerStyle}">
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="30" />
+                                <ColumnDefinition Width="*" />
+                            </Grid.ColumnDefinitions>
+
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
+
+                            <Grid Grid.Row="0"
+                                  Grid.RowSpan="2"
+                                  VerticalAlignment="Center">
+                                <Border Name="avatarMask" Background="White" CornerRadius="3" Width="20" />
+
+                                <StackPanel>
+                                    <StackPanel.OpacityMask>
+                                        <VisualBrush Visual="{Binding ElementName=avatarMask}" />
+                                    </StackPanel.OpacityMask>
+
+                                    <Image 
+                                        Height="30"
+                                        Width="30"
+                                        RenderOptions.BitmapScalingMode="HighQuality"
+                                        Source="/GitHub.VisualStudio;component/Resources/default_user_avatar.png"/>
+                                </StackPanel>
+                            </Grid>
+
+                            <TextBlock Text="I am an example commit message"
+                                       ToolTip="I am an example commit message"
+                                       VerticalAlignment="Top"
+                                       Foreground="{DynamicResource GitHubVsToolWindowText}"
+                                       FontSize="14"
```

Comment:
Instead of this being hardcoded, is there a way to set this relative to the user's font settings?